server/svix-server/src/queue/redis.rs

```diff
@@ -41,70 +41,60 @@ const MAIN: &str = "{svix_queue}_main";
 const PROCESSING: &str = "{svix_queue}_processing";
 const DELAYED: &str = "{svix_queue}_delayed";
```

Comment:
Nitpick, but I'd change these to be: `{queue}_svix_main`.
I'm not entirely sure how the sharding works in redis and this feels more safe. With this the key is surely unique, and there's still a sharding key.